### PR TITLE
✅ Improve tests cases

### PR DESCRIPTION
**Pull request by iurynogueira**

### CODE REVIEW

File: frontend/src/components/trail-list/trails-list.test.tsx [hunk 1]
```diff
@@ -55,4 +55,11 @@ describe('TrailList', () => {
       screen.getByText(`${trail.surveys_quantity} Survey`)
     ).toBeInTheDocument();
   });
+
+  it('should open a specific trail', async () => {
+    render(<TrailList data={trails} />);
+    const trailItem = screen.getByText(trails[0].name);
+    fireEvent.click(trailItem);
+    expect(trailItem).toBeInTheDocument();
+  });
```
Comment (andersonmalheiro) [inline frontend/src/components/trail-list/trails-list.test.tsx:64]: I think this test isn't doing what it's supposed to do. 

I suggest you to put move the current expect  before the `fireEvent` call and add another expect after the click to check if something new occurred with the execution of the event.

**Review by andersonmalheiro — CHANGES_REQUESTED** [review]

File: frontend/src/components/trail-list/trails-list.test.tsx (hunk 1 above)
Comment (iurynogueira) [reply frontend/src/components/trail-list/trails-list.test.tsx:64]: How we can check if a new page open?

Comment (andersonmalheiro) [reply frontend/src/components/trail-list/trails-list.test.tsx:64]: you can mock the react-router method and check if it's called with the expected URL

Comment (iurynogueira) [reply frontend/src/components/trail-list/trails-list.test.tsx]: Fixed, thanks @andersonmalheiro !

**Pull request merged**